models/deaths-conv-cases/update-conv.R

```diff
@@ -15,21 +15,19 @@ target_date <- readRDS(here("data", "target_date.rds"))
 # assumes no missing data (either explicit or implicit)
 source(here("utils", "get-us-data.R"))
 # get observed deaths
-deaths_raw <- get_us_deaths(data = "daily")
+deaths_raw <- get_us_data("deaths", 
+                          include_national = TRUE,
+                          incident = TRUE)
 deaths_raw <- as.data.table(deaths_raw)
-observations <- deaths_raw[, .(region = state, date, secondary = deaths)]
-us_obs <- copy(observations)[, .(secondary = sum(secondary, na.rm = TRUE)), by = "date"]
-us_obs <- us_obs[, region := "US"]
-observations <- rbindlist(list(us_obs, observations), use.names = TRUE)
+observations <- deaths_raw[, .(region = state, date, secondary = value)]
 
 # get observed cases
-cases <- get_us_cases(data = "daily")
+cases <- get_us_data("cases", 
```

Comment:
Comment rather than change. This is a US repo so arguably could call this function `get_data`

Reply:
Agreed, if setting it up today I would do that, but was keeping changes to a minimum.